robotoff/off.py

```diff
@@ -125,7 +125,8 @@ def get_api_product_url(server: Union[ServerType, str]) -> str:
 def get_base_url(server: Union[ServerType, str]) -> str:
     if isinstance(server, str):
         server = server.replace("api", "world")
-        return "https://{}".format(server)
+        scheme = settings.BaseURLProvider().scheme
```

Comment:
It would be good to add a comment to explain why this is useful here (to be able to work with a local Product Opener in dev mode)